localstack/services/install.py

```diff
@@ -238,8 +238,9 @@ def download_and_extract():
 
     try:
         download_and_extract()
-    except Exception:
+    except Exception as e:
         # try deleting and re-downloading the zip file
+        LOGGER.error('download_and_extract_with_retry caught exception: %s' % e)
```

Comment:
Can we combine the two log messages into one (with log level `info` or `warning`)? Something like
```
LOGGER.info('Unable to extract file, re-downloading ZIP archive %s: %s' % (tmp_archive, e))
```